common/utils.cpp

```diff
@@ -685,5 +685,185 @@ std::string getCurrentSystemTime()
     return ss.str();
 }
 
+std::vector<std::vector<uint8_t>> findAllStateEffecterPDRs(uint8_t /*tid*/,
```

Comment:
getStateEffecterPDRsByType makes more sense right ?

Reply:
std::vector<std::vector<uint8_t>> -> use pldm types to make it implicit to understand what you are trying to return.

Reply:
done